Packs/Zscaler/Integrations/Zscaler/Zscaler.py

```diff
@@ -113,8 +115,7 @@ def http_request(method, url_suffix, data=None, headers=None, num_of_seconds_to_
                     raise Exception('Your request failed with the following error: {}.\nMessage: {}'.format(
                         res.status_code, res.text))
     except Exception as e:
-        LOG('Zscaler request failed with url={url}\tdata={data}'.format(url=url, data=data))
-        LOG(e)
+        return_error(str(e))
```

Comment:
Are both the `LOG` calls being handled by the single `return_error` call?
It looks like `return_error` is printing the exception, don't we need to format the error message as:
```
LOG('Zscaler request failed with url={url}\tdata={data}'.format(url=url, data=data))
```

Reply:
obviously not. But I felt like showcasing url and payload to end user is not a good idea, hence removed that.